### ISSUE TITLE
Military Behemoths

### ISSUE THREAD

**Issue by captaincobbs** [captaincobbs/ESG-Mod#1091]
Make Behemoth Fleet-wide versions of the following coordinator modules:
- Shield Regenerators
- EMP Disruption Field
- Flotilla Shield
- Lock-On
- Shrapnel

New Modules
- Fleet Accuracy (+10% Accuracy on Fleet at a certain range?)
- Ship Cost Reduction on Orbited System
- Fleet Health
- Fleet Damage
- Fleet Evasion
- Repair Fleet to Full after Battle (one per Fleet)
- +1 Action Point after Battle (one per Fleet)
- Improved BEB
- % Damage Reduction on Behemoth

=====================
Added in #1092 
Military Behemoth Changes
- Military Behemoth CP reduced to 5
- Weapon Arcs increased to 270

These are just the potential changes I thought were possible, lmk what feedback you have

**Comment by captaincobbs** [comment]
Waiting on @stygmaskyzoo for icons

**Comment by captaincobbs** [comment]
Behemoth Weaponization
~~Fleet Repair Module~~ (100% Repair after Battle, +25% Repair each Phase, +50% Movement back after Battle cannot stack)
Cost: 250 Industry, 2 Hyperium
Ship Cost Reduction on Orbited System Module (10% reduction, cannot stack on system)
Cost: 250 Industry, 5 Titanium 5 Hyperium
~~Fleetwide Shrapnel Module~~ (100% crew damage, 50% boarding pod damage, cannot stack)
Cost: 250 Industry, 2 Titanium

Juggernaut Blueprints
~~Fleetwide Accuracy Module~~ (+10% Accuracy, cannot stack)
Cost: 250 Industry, 5 Antimatter
Fleetwide Health Bonus (+25% Health, cannot stack)
Cost: 250 Industry, 5 Adamantium
Fleetwide Damage Bonus (+25% Damage, cannot stack)
Cost: 250 Industry, 5 Antimatter
Fleetwide Evasion Bonus (+10% Evasion, cannot stack)
Cost: 250 Industry, 5 Adamantium

N-Dimensional Industry
Fleetwide Lock-On Module (8% on Fleet, cannot stack, cannot combine with coordinator module)
Cost: 250 Industry, 10 Antimatter

Advanced Fusion Power / Hyperium Magnetics
~~Action Point Module~~ (+1 Action Point max on fleet, +1 Action Points gained after battle, cannot stack)
Cost: 250 Industry, 1 Titanium, 1 Hyperium

Juggernaut Module Upgrade
Upgraded Blast Effect Battery (Same as standard BEB, but 450 damage)
Cost: 250 Industry, 1 Antimatter, 1 Adamantium

Distributed Fusion
Fleetwide EMP Disruption Field (-75% Shield and Weapon EMP Deactivation Duration on Fleet, -100% Shield and Weapon EMP Deactivation Duration on Self, cannot stack)
Cost: 250 Industry, 2 Antimatter, 2 Adamantium

Vacuum Protection
Fleetwide Shield Regeneration (200 Shield per Second on Fleet, cannot stack, scales with module multiplier)
Cost: 250 Industry, 3 Antimatter

Fleet Shielding
Hyperium Fleet Shield (1000 Shield Health, 75 Shield Absorption, cannot stack)
Cost: 250 Industry, 2 Hyperium

Antimatter Channeling
Antimatter Fleet Shield (2000 Shield Health, 250 Shield Absorption, cannot stack)
Cost: 250 Industry, 2 Antimatter

**Comment by stygmaskyzoo** [comment]
are those strategics definitive? 
making the ones with two colors is gonna be fun

**Comment by captaincobbs** [comment]
Yes, though they may be modified after this is finished due to balance reasons.

**Comment by stygmaskyzoo** [comment]
updated most of the monocolored modules:
![module + behemoth + shrapnel + recolor](https://user-images.githubusercontent.com/73248181/170894473-dd3cf2ec-e895-48ba-9235-ea17b011e728.png)
![module + behemoth + accuracy + recolor](https://user-images.githubusercontent.com/73248181/170894475-1bb4234a-cd4a-4394-9ca1-be0e9043900a.png)
![module + behemoth + damage + recolor](https://user-images.githubusercontent.com/73248181/170894476-36e531f7-e45a-4368-888f-66fecb7825e8.png)
![module + behemoth + evasion + recolor](https://user-images.githubusercontent.com/73248181/170894477-49376977-b733-4e40-bbef-207f7241616a.png)
![module + behemoth + health + recolor](https://user-images.githubusercontent.com/73248181/170894478-2a16b89c-0ee9-4675-9927-4bd61850e9c7.png)
![module + behemoth + lock on + recolor](https://user-images.githubusercontent.com/73248181/170894480-453199e2-e198-4a03-adb5-03fca87f6722.png)
![module + behemoth + repair + recolor](https://user-images.githubusercontent.com/73248181/170894481-45278d71-d71e-4688-b0f2-55ef947b7f2d.png)
![module + behemoth + shield regen + recolor](https://user-images.githubusercontent.com/73248181/170894482-47936254-b793-41d7-9e28-2b7089572aae.png)


are these last ones the same icon with different strategics?

Fleet Shielding
Hyperium Fleet Shield (1000 Shield Health, 75 Shield Absorption, cannot stack)
Cost: 250 Industry, 2 Hyperium

Antimatter Channeling
Antimatter Fleet Shield (2000 Shield Health, 250 Shield Absorption, cannot stack)
Cost: 250 Industry, 2 Antimatter

what happened to defense % only applied to behemoth?

**Comment by captaincobbs** [comment]
Yes, the last two are the same with different strategics. 

And you're right, oops, forgot defense %

Applied Cryogenics
Behemoth Lenser (25% Less Damage Taken on Ship, +20 Targeting Weight, cannot stack)
Cost: 160 Industry, 1 Antimatter

Split-second Ablatives
Behemoth Jammer (6 Movement Speed, -10 Targeting Weight, cannot stack)
Cost: 160 Industry, 1 Adamantian

**Comment by stygmaskyzoo** [comment]
![module + behemoth + emp + recolor](https://user-images.githubusercontent.com/73248181/170895848-8ea7ae96-d43b-4a70-9788-8ccfcb2a3e6f.png)
![module + behemoth + ship cost + recolor](https://user-images.githubusercontent.com/73248181/170895851-f3fda132-7c35-45c9-b48b-eb5191b2327c.png)
![module + action point + recolor](https://user-images.githubusercontent.com/73248181/170895853-62da0343-197c-4e48-8043-e0499aff74c9.png)
![module + behemoth + blast + recolor](https://user-images.githubusercontent.com/73248181/170895854-6038d0d4-cdcc-40bb-a4c2-663d9e6db041.png)

only jammer + lenser left
i'll have them soon, hopefully tomorrow

**Comment by NoProblemWithThisAcc** [comment]
on low contrast/brightness monitors the peach and red don't exactly show up like they should. Can you make the red a bit deeper and the orange a bit lighter so it is more easily distinguishable? This normally isn't a problem for single color icons but players might not expect the new mashup.

**Comment by stygmaskyzoo** [comment]
i've sent them to cobbs but anyway, here they are:
![module + behemoth + lenser](https://user-images.githubusercontent.com/73248181/171159717-106744df-af30-4182-9e0e-198b57b9728b.png)
![module + action point + recolor](https://user-images.githubusercontent.com/73248181/171159720-b809cde6-db57-490d-9bae-3e50d1c9fc86.png)
![module + behemoth + blast + recolor](https://user-images.githubusercontent.com/73248181/171159725-68fe665e-8961-441a-8407-ca9416a4ee2f.png)
![module + behemoth + emp + recolor](https://user-images.githubusercontent.com/73248181/171159728-99c5e48c-d66e-48d9-8ed2-f41b1f9f96bd.png)
![module + behemoth + jammer](https://user-images.githubusercontent.com/73248181/171159731-83657273-68e0-4dc9-bf1f-9d23e7ef9c5d.png)

**Comment by Arpharnator** [comment]
can't wait to muder people with these
